README.md

```diff
@@ -73,7 +73,7 @@ We appreciate all contributions. You can contribute in three ways:
 - Use PyKale in your project and let us know any bugs (& fixes) and feature requests/suggestions via creating an [issue](https://github.com/pykale/pykale/issues).
 - Contribute your code such as new features, functions, or examples via [fork and pull](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-collaborative-development-models) to become one of the [contributors](https://github.com/pykale/pykale/graphs/contributors).
```

Comment:
Not part of this PR, but do we prefer branches? Better to refer people to contributing guidelines for this?